src/stampy_nlp/settings.py

```diff
@@ -0,0 +1,30 @@
+import os
+from dotenv import load_dotenv
+
+
+load_dotenv()
+
+CODA_TOKEN: str = os.getenv('CODA_TOKEN')
+HUGGINGFACE_API_KEY: str = os.getenv('HUGGINGFACE_API_KEY')
+PINECONE_API_KEY: str = os.getenv('PINECONE_API_KEY')
+
+STAMPY_BUCKET: str = os.getenv('STAMPY_BUCKET', 'stampy-nlp-resources')
```

Comment:
this is unlikely to be changed, but it should make things easier to configure if it has to be deployed somewhere else